modules/reference/pages/rpk/rpk-transform/rpk-transform-init.adoc

```diff
@@ -0,0 +1,42 @@
+= rpk transform init
+:description: rpk transform init
+
+Initialize a transform.

+

+Creates a new transform using a template in the current directory.

+

+A new directory can be created by specifying it in the command, like:

+

+  rpk transform init foobar

+

+Will initialize a transform project in the foobar directory.
+
+== Usage
+
+[,bash]
+----
+rpk transform init [DIRECTORY] [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a]
+|===
+|*Value* |*Type* |*Description*
+
+|`-h, --help` |- |Help for init.
+
+|`--install-deps` |- |If dependencies should be installed for the project (default prompt).
+
+|`-l, --language` |string |The language used to develop the transform.
+
+|`--name` |string |The name of the transform.

```

Comment:
@rockwotj  should we even document this if there are no options at this point?

Reply:
Are you talking about the `--language` flag? Let's leave it in to show we will support others in the future, and I think that will come very soon.